ui/imports/shared/controls/TokenBalancePerChainDelegate.qml

```diff
@@ -55,6 +65,7 @@ StatusListItem {
             width: 16
             height: 16
             image.source: Style.svg("tiny/%1".arg(root.getNetworkIcon(chainId)))
+            visible: root.sensor.containsMouse ? index > d.indexesThatCanBeShown : true
```

Comment:
```suggestion
            visible: root.sensor.containsMouse && index > d.indexesThatCanBeShown
```

Reply:
I think I need to change it to 

            visible: !root.sensor.containsMouse || index > d.indexesThatCanBeShown